lib/treemap.c

```diff
@@ -0,0 +1,237 @@
+/*
+** Copyright (c) 2017-2018 Snowflake Computing, Inc. All rights reserved.
+*/
+
+#include <string.h>
+#include "treemap.h"
+#include "memory.h"
+
+/* 
+** Hashmap needs to be initialized
+** to a size which is sufficiently 
+** large and is a prime number
```

Comment:
I'm confused here. Here you say it needs to be initialized to a size that is sufficiently large and is a prime number, but `TREE_MAP_MAX_SIZE` is set to 1000 (which is large, but not prime)

Reply:
Stale comment from the hashmap implementation. Deleted.